kratos/python_scripts/csv_points_output_process.py

```diff
@@ -0,0 +1,60 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# other imports
+from KratosMultiphysics.multiple_points_output_process import MultiplePointsOutputProcess
+import csv, os
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+
+    params = settings["Parameters"]
+
+    default_settings = KratosMultiphysics.Parameters('''{
+        "help"              : "This process writes output for several points given in a .csv-file ('csv_file_path') to a file. Internally it holds an object of type MultiplePointsOutputProcess. Usage: The first line of the .csv-file should state the column names: 'x,y,z'. The following rows should each be one point with its x,y and z value in the respective column of the .csv-file, separated by a comma.",
+        "model_part_name"   : "",
+        "entity_type"       : "element",
+        "csv_file_path"         : "",
+        "output_variables"  : [],
+        "historical_value"  : true,
+        "search_tolerance"  : 1e-6,
+        "print_format"      : "",
+        "output_file_settings": {}
+    }''')
+
+    params.ValidateAndAssignDefaults(default_settings)
+
+    # retrieving file path and name
+    csv_file_path = params["csv_file_path"].GetString()
+    if not csv_file_path:
+        raise Exception('A file path has has to be provided!')
+
+    # initialize array for sampling points
+    points = []
+    p_count = 0
+
+    # open file and get points
+    with open(csv_file_path, newline=None) as file:
+        reader = csv.DictReader(file, delimiter=',')
+        for p_count, row in enumerate(reader):
+            points.append([float(row['x']), float(row['y']), float(row['z'])])
```

Comment:
```suggestion
        points = [(float(row['x']), float(row['y']), float(row['z'])) for row in reader]
```

better use list comprehension here, can be faster